addons/models/auth/AuthService.json

```diff
@@ -0,0 +1,125 @@
+{
+    "entityDefs": [
+        {
+            "name": "AuthService",
```

Comment:
Let's add Asset as a superType

Reply:
only possible if we also move Asset type creation here, till this point only base typeDefs are created